Sources/Private/CoreAnimation/ValueProviderStore.swift

```diff
@@ -125,13 +125,19 @@ extension AnimationKeypath {
       + keypath.keys.joined(separator: "\\.") // match this keypath, escaping "." characters
       + "$" // match the end of the string
 
-    // ** wildcards match anything
-    //  - "**.Color" matches both "Layer 1.Color" and "Layer 1.Layer 2.Color"
-    regex = regex.replacingOccurrences(of: "**", with: ".+")
-
-    // * wildcards match any individual path component
+    let doubleWildcardMarker = "DOUBLE"
+    let singleWildcardMarker = "SINGLE"
```

Comment:
what happens if someone has one of these tokens as a layer name—seems like we could accidentally treat it as a marker?

Reply:
Aaaaah thank you, good thinking. Yeah that would be a problem. I'll change this to something less likely to cause collisions. Maybe a UUID string?